### PR TITLE
[M] Code artifact

### PR DESCRIPTION
**Pull request by Bemhreth**
**test result**
Screen Shot 2015-01-21 at 12 10 26 PM

### CODE REVIEW

File: wao/notifier.py [hunk 1]
```diff
@@ -1,11 +1,9 @@
 from wao.brain_config import BrainConfig
 import requests
-import sys
 
-sys.path.append(BrainConfig.EXECUTION_PATH)
```
Comment (tef-github) [inline wao/notifier.py:5]: Can we remove EXECUTION_PATH from brain_config.py

**Review by tef-github — CHANGES_REQUESTED** [review]
CR

**Comment by tef-github** [comment]
@Bemhreth don't we need to create the directories that were migrated from execution to here manually ?
the ones that start with underscore

**Comment by tef-github** [comment]
@Bemhreth Good job! I don't have many code comments on this one. Let's wait till all prs are ready + artifact is stable (without bugs) and we will merge them together at the same time.

**Comment by Bemhreth** [comment]
> @Bemhreth don't we need to create the directories that were migrated from execution to here manually ? the ones that start with underscore

we have create directory for those files so those functions will create the directories needed but if there is any folder that doesn't have one I will search and create one for them.

File: wao/brain_config.py [hunk 2]
```diff
@@ -1,23 +1,27 @@
 class BrainConfig:
+    BRAIN = "Freq_Cluc6werk"
```
Comment (tef-github) [inline wao/brain_config.py]: add a comment //set from strategy files

File: wao/wao_strategy_controller.py [hunk 3]
```diff
@@ -3,17 +3,15 @@
...
-sys.path.append(BrainConfig.EXECUTION_PATH)
-from config import Config
+from execution.config import Config
 
 
 class WAOStrategyController:
 
     def __init__(self, brain, time_out_hours, dup):
         self.brain = brain
+        BrainConfig.BRAIN = brain
```
Comment (tef-github) [inline wao/wao_strategy_controller.py]: set this in strategy files and remove self.brain from strategies and WaoStrategyController

remove all references and parameters

wherever it's used at the end, replace by BrainConfig.BRAIN

**Review by tef-github — CHANGES_REQUESTED** [review]
cr

**Review by tef-github — APPROVED** [review]
LGTM, let's wait before merge

**Review by tef-github — CHANGES_REQUESTED** [review]
This is APPROVED. Waiting for google sheet records.

**Review by tef-github — APPROVED** [review]

**Pull request merged**